### PR TITLE
Import csv

### PR DESCRIPTION
**Pull request by Duez**
ajout d'un module d'export/import de fichier .csv (db/csv.py) au format du google doc
plus quelques scripts stand alone(scripts/import_csv scripts/export_csv)

### CODE REVIEW

File: icubam/db/csv.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+	def __init__(self, db):
+		self.store = db
+
+	def get_default_admin(self):
+		# create default admin
+		return self.store.add_default_admin()
+
+	def import_icus(self,admin_user_id: int, csv_file_path:str, forceUpdate=False):
+		csv_data = csv.reader(open(csv_file_path))
```
Comment (rth) [inline icubam/db/csv.py:15]: any reason not to use pandas.read_csv? That would generally be faster for processing that for loops over rows.

File: icubam/db/csv.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+class CSV:
+	""" """
+
+	def __init__(self, db):
+		self.store = db
+
+	def get_default_admin(self):
+		# create default admin
+		return self.store.add_default_admin()
```
[the remark this answers is not in the document]
Comment (dulacarnold) [reply icubam/db/csv.py:12]: Yes, there is a .style.yapf file.  Also we are on a 2-spaces indentation standard.

File: icubam/db/csv.py (hunk 1 above)
Comment (dulacarnold) [reply icubam/db/csv.py:15]: +1 for a pandas-based solution for maintainability down the line.

File: icubam/db/csv.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (tgy) [reply icubam/db/csv.py:12]: happy to see it now serves a purpose.

File: icubam/db/csv.py (hunk 1 above)
Comment (tgy) [reply icubam/db/csv.py:15]: plus we already have pandas as a dependency

File: icubam/db/csv.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Duez) [reply icubam/db/csv.py]: this should be ok now

File: icubam/db/csv.py (hunk 1 above)
Comment (Duez) [reply icubam/db/csv.py]: i don't know panda(to be honest i don't know too much of python either) I will give it a look

**Comment by Duez** [comment]
> Thanks! If would be necessary to add unit tests for the added functionality. e.g. under `icubam/db/csv_test.py` (see other test files in the repo)

added some tests

File: icubam/db/csv.py (hunk 1 above)
Comment (tgy) [reply icubam/db/csv.py]: simply use `pd.read_csv(csv_file_path)` here instead

Comment (tgy) [reply icubam/db/csv.py]: and `import pandas as pd`

Comment (Duez) [reply icubam/db/csv.py]: Done, it was a bit more than just a line, since iterating trough dataframe is done differently but in the end it's cleaner.

**Pull request merged**